Examples/Examples/FormViewExampleView.swift

```diff
@@ -72,40 +61,39 @@ struct FormViewExampleView: View {
                     attributionBarHeight: attributionBarHeight,
                     selectedDetent: $detent,
                     horizontalAlignment: .leading,
-                    isPresented: $isFormPresented
+                    isPresented: $model.isFormPresented
                 ) {
-                    FormView(featureForm: featureForm)
-                        .padding([.horizontal])
+                    if let featureForm = model.featureForm {
+                        FormView(feature: featureForm.feature, featureForm: featureForm)
+                            .padding([.horizontal])
+                    }
```

Comment:
I tried to figure out a way to make the model's `featureForm` property non-nullable, but because  `featureForm` is transient, it wasn't possible. So I'd prefer to keep the `if let` in place, just to be specific, even though it *should* always be non-nil when `model.isFormPresented == true`.